src/miner_lora.erl

```diff
@@ -72,8 +73,16 @@ init(Args) ->
     UDPIP = maps:get(radio_udp_bind_ip, Args),
     UDPPort = maps:get(radio_udp_bind_port, Args),
     {ok, Socket} = gen_udp:open(UDPPort, [binary, {reuseaddr, true}, {active, 100}, {ip, UDPIP}]),
+    MirrorSocket = case application:get_env(miner, radio_mirror_port, undefined) of
+        undefined ->
+            undefined;
+        P ->
+            {ok, S} = gen_udp:open(P, [binary, {active, true}]),
```

Comment:
This mirror socket is not being closed in the terminate.  
In addition the process itself is not trapping exits and thus risk terminate and its cleanup not being called

Reply:
The socket will close when the controlling process closes, no?

Reply:
That would be the expectation but with sockets i think its always good practise to explicitly close them and minimise any potential oddities that could lead to a leak

Reply:
I think we can rely on BEAM closing the socket when the controlling process dies, that's pretty much guaranteed beyond a BEAM bug.